Remove readme step from policy creation - add slide-in for readme

jordojordo opened the pull request:
Fix #775 

This will remove the policy readme step and replaces it with a slide out component showing the readme.

I also updated the ordering of the configuration tabs, and removed the module input if the policy is not custom.


https://github.com/rancher/kubewarden-ui/assets/40806497/c705361c-a450-440d-8fc6-eadbb1ff50e9

aalves08 reviewed (approved):
Codewise, LGTM

viccuad reviewed (approved):
LGTM for what is worth

kravciak commented:
PR works as expected, but I think removing module input is a problem.

With this change I can't see current policy version and I can't update it from UI.

jordojordo commented:
> I think removing module input is a problem.
> With this change I can't see current policy version and I can't update policy from UI.

You can still see and update the module from the yaml view.

> Also Show More button would be useful on policy edit page in addition to create page.

Good point, I added the Show More button to the policy edit page. Also I updated the wording to "Show Readme".
